clients/fides-js/src/lib/cookie.ts

```diff
@@ -302,11 +283,43 @@ export const updateExperienceFromCookieConsent = ({
               Boolean(cookieConsent[item.id]),
               ConsentMechanism.OPT_IN
             )
-          : undefined;
+          : // if experience contains a tcf entity not defined by tcfEntities, we override experience current pref with the default pref
+            item.default_preference;
```

Comment:
I know we talked about this live, but it worries me a little. it's okay right now since all of our default preferences are "opt out", but if you imagine that one became "opt in", this might pose a problem. for example, if Vendor A was default "opt in", but not included in the `fidesString`, this means either:
1. the user opted out of Vendor A (since `fidesString` only saves opt ins)
2. the user never saw Vendor A
this poses a problem in case 1️⃣ because the user may have actually opted out, but now, since we don't see it in the string, we opt them in

again, this won't happen right now because everyone's default preference is "opt out" but I feel like it could bite us one day without us realizing it

Reply:
okay, thanks for looking into that!